lblod/frontend-reglementaire-bijlage
# Made the besluit template able to be instantiated

## Overview
Modified the besluit template so it can be instantiated by the template-uuid-instatiator library

### connected issues and PRs:
https://github.com/lblod/frontend-gelinkt-notuleren/pull/724


### How to test/reproduce
Create a template based on a besluit and observe the uuid is of the type of `http://data.lblod.info/id/besluiten/--ref-uuid4-${decisionUuid}`

### Challenges/uncertainties
None, I double checked the besluit base uri



### Checks PR readiness
